### PR TITLE
refactor(dapp): remove listings from redux reducers/actions

### PR DESCRIPTION
**Pull request by nickreynolds**

### CODE REVIEW

File: packages/sdk/src/react/boosts/payments/BoostPayFormEth.tsx [hunk 1]
```diff
@@ -146,6 +146,7 @@ export class BoostPayFormEth extends React.Component<BoostPayFormEthProps, Boost
   };
 
   private handleTransactionHash = async (txHash: TxHash) => {
+    console.log("handle transaction hash. txHash: ", txHash);
```
Comment (tobek) [inline packages/sdk/src/react/boosts/payments/BoostPayFormEth.tsx]: I assume these were left in by accident.

**Comment by nickreynolds** [comment]
we already removed the graphql toggle a while ago, and removed most non-graphql flows, and yeah will remove the flag from redux entirely soon

**Comment by dankins** [comment]
:tada: This PR is included in version 4.8.31 :tada:

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Pull request merged**